packages/compass-user-data/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@mongodb-js/compass-user-data",
+  "productName": "compass-user-data Plugin",
```

Comment:
```suggestion
```

Reply:
I am sorry, I can't figure out a change here.

Reply:
Sorry: `productName` is only needed for plugins, this is just a library package, there is a bug in create workspace that nobody can get to to fix this so far, so I'm just suggesting you remove it manually

Reply:
aha, got it.